lib/git_reflow/git_server/git_hub/pull_request.rb

```diff
@@ -63,7 +63,11 @@ def comments
         end
 
         def last_comment
-          "#{comments.last.body.inspect}"
+          if comments.nil? or comments.last.nil?
```

Comment:
@pboling, the current implementation of the `comments` method already returns an emtpy `array` by default, but I was getting a NilException when the array is empty, since `comments.last` is nil and so `"#{comments.last.body.inspect}"` was throwing an exception.

Since each method is a separate entity, just to make sure that we don't run into the issue again, I added:
1. `comments.nil?`
2. `comments.last.nil?`
Just in case another method uses `last_comment`, we won't get a NilException.